edumanage/forms.py

```diff
@@ -97,8 +100,10 @@ def clean_acct_port(self):
 
     def clean_rad_pkt_type(self):
         rad_pkt_type = self.cleaned_data['rad_pkt_type']
-        institution = self.cleaned_data['instid']
-        if institution.ertype in [1,3]:
+	if not 'ertype' in self.cleaned_data:
+                raise forms.ValidationError(_('The Type field is required to validate this field.'))
+        ertype = self.cleaned_data['ertype']
+        if ertype in [1,3]:
```

Comment:
Sorry, scratch this comment, it is not true: there is no use of `rad_pkt_type` for SP only servers/institutions. Your approach is fine and it is more appropriate than checking institution `ertype`. Eventually we should also remove/hide this field from the form for such cases.